packages/admin-panel/src/table/actions.js

```diff
@@ -90,7 +90,7 @@ const refreshDataWithDebounce = debounce(
     const sortString = JSON.stringify(sortObjects);
 
     // Set up columns
-    const columnSources = [...columns.map(column => column.source).filter(source => source), 'id'];
+    const columnSources = columns.map(column => column.source).filter(source => source);
```

Comment:
Admittedly a micro-optimisation that doesn’t matter with this small an array, but I would generally tend to filter before mapping—less stuff to map this way

Probably not worth changing in this particular instance